### DISCUSSION TITLE
Use with DSC?

### DISCUSSION THREAD

**Discussion by LeoKurz68** [PSAppDeployToolkit/PSAppDeployToolkit#1044] [Ideas]
Anyone ever used PSAppDeplyToolkit with Powershell DSC?

__Leo

**Comment by mjr4077au** [comment]
@LeoKurz68 just wondering if you made any attempts here since August last year? I'd like to think our v4 module is more readily usable with DSC, but I've not heard of anyone doing anything specifically with DSC and PSAppDeployToolkit (yet).
